tests/integration/components/public/ticket-list-test.js

```diff
@@ -1,14 +1,50 @@
 import { test } from 'ember-qunit';
+import Ember from 'ember';
 import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
 import hbs from 'htmlbars-inline-precompile';
 
-moduleForComponent('public/ticket-list', 'Integration | Component | public/ticket list');
+const { Object, ArrayProxy } = Ember;
```

Comment:
No need array proxy. You can just use `A`
```js
const { Object, A } = Ember;
```

And also, alias `Object` to `EmberObject` to avoid confusion with the native js `Object`

```js
const { Object: EmberObject, A } = Ember;
```